server/src/main/java/org/opensearch/index/remote/RemoteStoreUtils.java

```diff
@@ -69,4 +75,31 @@ public static String getSegmentName(String filename) {
 
         return filename.substring(0, endIdx);
     }
+
+    /**
+     *
+     * @param mdFiles List of segment/translog metadata files
+     * @param fn Function to extract PrimaryTerm_Generation and Node Id from metadata file name .
+     *          fn returns null if node id is not part of the file name
+     */
+    static public void verifyNoMultipleWriters(List<String> mdFiles, Function<String, Tuple<String, String>> fn) {
```

Comment:
nit: public static order

Reply:
Will take that in another PR to save time on gradle builds.